pkg/kubelet/network/cni/cni_test.go

```diff
@@ -42,10 +42,15 @@ import (
 )
 
 // The temp dir where test plugins will be stored.
-const testNetworkConfigPath = "/tmp/fake/plugins/net/cni"
-const testVendorCNIDirPrefix = "/tmp"
+func tmpDirOrDie() string {
+	dir, err := ioutil.TempDir(os.TempDir(), "cni-test")
+	if err != nil {
+		panic(fmt.Sprintf("error creating tmp dir: %v", err))
+	}
+	return path.Join(dir, "plugins", "net", "cni")
```

Comment:
might be better to have this just return `dir`, then call this function only once in TestCNIPlugin, using it to construct both other directories.

then the cleanup only needs to worry about this one root directory, too.